opts/parse_test.go

```diff
@@ -0,0 +1,73 @@
+package opts
+
+import (
+	"testing"
+
+	"gotest.tools/v3/assert"
+	"gotest.tools/v3/env"
+	"gotest.tools/v3/fs"
+)
+
+func TestReadKVEnvStrings(t *testing.T) {
+	emptyEnvFile := fs.NewFile(t, t.Name())
+	defer emptyEnvFile.Remove()
+
+	// EMPTY_VAR should be set, but with an empty string as value
+	// FROM_ENV value should be substituted with FROM_ENV in the current environment
+	// NO_SUCH_ENV does not exist in the current environment, so should be omitted
+	envFile1 := fs.NewFile(t, t.Name(), fs.WithContent("Z1=z\nEMPTY_VAR=\nFROM_ENV\nNO_SUCH_ENV"))
```

Comment:
nit: use backquote notation and "real" indentation instead of `\n` ?

Reply:
Thought this was slightly shorter, but I can change yes (I dislike the literals a bit if they use multiple lines as it messes up indentation of the code 😂)

Reply:
Yes same here... but I think it's still more readable than `KEY=VALUE\nKEY=VALUE\n`

Reply:
and it's just nit picking BTW 😄